website/docs/history-and-hard-forks/intro-to-hard-fork.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: intro-to-hard-fork
+title: "Intro to Hard Fork"
+---
+
+# What is a Hard Fork?
```

Comment:
shall we use a ## for sub titles? right now there is no table of content in the left